app/mailers/user_mailer.rb

```diff
@@ -46,27 +46,12 @@ def second_upload_reminder(submission:, user:, user_detail:)
     @user_detail = user_detail
     @utm_params =
       utm_params(
-        source: 'drip-consignment-reminder-e02',
+        source: 'drip-consignment-reminder-e02-v2',
```

Comment:
Updating the utm-source to reflect that this second reminder is different than it used to be.